requirements.txt

```diff
@@ -1,67 +1,33 @@
 # keyring is required to provide GitHub token to EasyBuild;
-# keyring v5.7.1 is last version to be compatible with py2.6;
 # for recent versions of keyring, keyrings.alt must be installed too
-keyring==5.7.1; python_version < '2.7'
-keyring<=9.1; python_version >= '2.7'
-keyrings.alt; python_version >= '2.7'
+# 19.0 dropped Python 2 support
+keyring<19.0; python_version < '3.0'
+keyring; python_version >= '3.0'
+keyrings.alt
 
-# GitDB 4.0.1 no longer supports Python 2.6
-gitdb==0.6.4; python_version < '2.7'
-gitdb; python_version >= '2.7'
```

Comment:
Hmm, is gitdb not needed at all any longer?

Reply:
It is an indirect dependency of something. So not on of ours and doesn't need to be listed here. We only had it due to the python_version requirement